src/org/traccar/database/DataManager.java

```diff
@@ -521,6 +521,31 @@ public void updateLatestPosition(Position position) throws SQLException {
                 .executeQuery(Position.class);
     }
 
+    //added by Erez
+    public void clearPositionsHistory() throws SQLException {
+        //SimpleDateFormat s = new SimpleDateFormat("yyyy-MM-dd");
+        String histDays = config.getString("database.positionsHistoryDays");
```

Comment:
ok, I found how to do this in the code

On Wed, Jul 13, 2016 at 9:39 AM, Ninio Erez <ninioe@gmail.com> wrote:

> how do I use the getInteger, can you give me an example?
>
> Integer.getInteger(histDays) cause an exeption
>
>
> On Wed, Jul 13, 2016 at 1:48 AM, Anton Tananaev <notifications@github.com>
> wrote:
>
>> In src/org/traccar/database/DataManager.java
>> <https://github.com/tananaev/traccar/pull/2104#discussion_r70536620>:
>>
>> > @@ -521,6 +521,31 @@ public void updateLatestPosition(Position position) throws SQLException {
>> >                  .executeQuery(Position.class);
>> >      }
>> >
>> > +    //added by Erez
>> > +    public void clearPositionsHistory() throws SQLException {
>> > +        //SimpleDateFormat s = new SimpleDateFormat("yyyy-MM-dd");
>> > +        String histDays = config.getString("database.positionsHistoryDays");
>>
>> Use getInteger instead.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/tananaev/traccar/pull/2104/files/d045c1e2174146ddc8ad8b2ef4442863f3d84a11#r70536620>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AH9fFxlXC48_2YS79mEKzoeYgT6TJ01cks5qVBmjgaJpZM4JKXy->
>> .
>>
>
>